src/components/collapsible/Collapsible.js

```diff
@@ -119,9 +141,15 @@ class Collapsible extends React.Component<Props, State> {
                     </PlainButton>
                     {isOpen && headerActionItems}
                 </div>
-                <AnimateHeight duration={animationDuration} height={isOpen ? 'auto' : 0}>
-                    <div className="collapsible-card-content">{children}</div>
-                </AnimateHeight>
+                <div
+                    ref={this.ref}
+                    style={{
+                        overflow: 'hidden',
+                        transition: `height ${animationDuration}ms cubic-bezier(0.4, 0, 0.2, 1) 0ms`,
```

Comment:
Probably briefly test IE11 too, so that things arent janky

Reply:
https://cloud.box.com/s/z38hzyup9xmxcpls3f5fec7oarbk2ubg